images/opentelemetry/cloudbuild.yaml

```diff
@@ -16,4 +16,5 @@ steps:
     - -c
     - |
       gcloud auth configure-docker \
-      && cd images/ && make NAME=opentelemetry push
+      && cd images/opentelemetry && make NGINX_VERSION=1.25.3 push \
+      && make NGINX_VERSION=1.21.6 push
```

Comment:
right now we will build OTEL standalone just for v1.21.6. v1.25.3 will have it bundled in